src/compiler/createWebSocketServer.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ * 
+ * @flow
+ */
+
+const xpipe = require('xpipe');
+const http = require('http');
+const { Server } = require('ws');
+const fs = require('fs');
+
+/**
+ * Create WebSocket server using HTTP server over Unix socket.
+ */
+module.exports = function createWebSocketServer() {
+  const socketAddress = xpipe.eq(`/tmp/HAUL_SOCKET_.socket`);
```

Comment:
Can we change it to a unique address? We should be able to run multiple instances of Haul with different ports. It's also necessary for expo integration.